.github/workflows/CI.yml

```diff
@@ -0,0 +1,24 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.0', '1.4', 'nightly']
```

Comment:
Does GitHub action support `'1'` for the latest release?

Reply:
If I remember correctly it did not work when I tried. But let me try again.

Reply:
Yes, works.

Reply:
I changed it to use `['1.0', '1', 'nightly']`